builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -90,13 +90,15 @@ node {
 
 			if (configObj.service_id) {
 				service_config = serviceMetadataLoader.loadServiceMetadata(configObj.service_id)
+				service_config << configObj
```

Comment:
What is the purpose of this?

Reply:
As discussed to make a consistency with other workflows and expecting if any future use.